setup.py

```diff
@@ -54,11 +52,14 @@
         'Operating System :: Microsoft :: Windows',
         'Operating System :: POSIX',
         'Operating System :: Unix',
+        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Implementation :: CPython',
```

Comment:
I do not use any C in this library.

Reply:
No, but it supports running on [CPython](https://en.wikipedia.org/wiki/CPython) and maybe not [Jython](https://en.wikipedia.org/wiki/Jython) or [PyPy](https://en.wikipedia.org/wiki/PyPy).

https://pypi.python.org/pypi?%3Aaction=list_classifiers

Let me know if you want that removing.

Reply:
Please remove.